File: Blog/client/src/app/models/post.ts
export interface Post {
  id?: number;
  title: string;
  permaLink: string;
  category: {
    id: string;
    name: string;
  };
  heroImg: string;
  excerpt: string;
  content: string;
  isFeatured: boolean;
  views: number;
  status: string;
  createdAt: Date;
}

File: Blog/server-clean-arc/Blog.Domain/Post.cs
using Blog.Domain.Common;

namespace Blog.Domain
{
    public class Post : BaseDomainEntity
    {
        public string Title { get; set; }
        public string PermaLink { get; set; }
        public Guid CategoryId { get; set; }
        public Category Category { get; set; }
        public string HeroImg { get; set; }
        public string Excerpt { get; set; }
        public string Content { get; set; }
        public bool IsFeatured { get; set; } = false;
        public int Views { get; set; } = 0;
    }
}
